lib/getConfig.js

```diff
@@ -10,17 +11,51 @@ const getConfig = (altPath) => {
   const configObject = json.readToObjSync(pathString);
   const globalConfig = json.readToObjSync(path.join(homedir, '.sgcrc'));
   const packageConfig = json.readToObjSync(path.join(cwd, 'package.json')).sgc;
-  const sgcrcDefaultConfig = json.readToObjSync(path.join(__dirname, '..', '.sgcrc_default'));
+  const sgcrcDefaultConfig = json.readToObjSync(path.join(__dirname, '..', '.sgcrc'));
+  const sgcrcTestDefaultConfig = json.readToObjSync(path.join(__dirname, '..', '.sgcrc_default'));
```

Comment:
As we discussed before. It is to have a 'global' config in the test cases. Maybe you have a better solution for that.